scripts/extract_schemas.py

```
@@ -0,0 +1,147 @@
+Extract inline schemas from plex-api-spec.yaml into reusable components.
+Run from repo root: python scripts/extract_schemas.py
+"""
+
+import yaml
+from copy import deepcopy
+
+SPEC_PATH = "plex-api-spec.yaml"
+
+
+def load_spec():
+    with open(SPEC_PATH, "r") as f:
```

Ruff (0.15.15)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)